src/transformers/sinon.ts

```diff
@@ -698,15 +713,48 @@ function transformMockTimers(j, ast) {
     })
 }
 
+/* 
+  checks if file originally had jsdom pragma, and ensure it's re-added
+  if it was removed
+*/
+function handleJSDomPragma(j, ast, filepath) {
```

Comment:
I believe this makes sense for everyone to keep. I'm wondering in which case this was removed?

The test case `keeps @jest-environment jsdom` is successful even if `keepJestEnvironment` is set to `false`.

Reply:
I think it's this problem, but we may be able to handle in a more generic way:
https://github.com/facebook/jscodeshift/blob/main/recipes/retain-first-comment.md

Reply:
However, I just found that the code is using:
https://github.com/skovhus/jest-codemods/blob/main/src/utils/imports.ts#L107
https://github.com/skovhus/jest-codemods/blob/main/src/utils/imports.test.ts#L190
which follows that recipe from jscodeshift, so this likely isn't necessary either.